CHANGELOG.md

```diff
@@ -20,14 +20,31 @@ This release has the following breaking changes:
   - The ZIP-317 convention fee increases based on the number of logical transparent or
     shielded actions in a transaction.
   - This change has no impact under normal network conditions.
-- TODO: search the changelog for breaking changes
 
 ### Security
 
 - Fix a rare crash and memory errors when Zebra's RPC server shuts down ([#5591](https://github.com/ZcashFoundation/zebra/pull/5591))
 - Evict transactions from the mempool using the ZIP-317 conventional fee ([#5703](https://github.com/ZcashFoundation/zebra/pull/5703))
 
-TODO: the rest of the changelog
+### Added
+
+- Add submitblock RPC method ([#5526](https://github.com/ZcashFoundation/zebra/pull/5526))
+
+### Changed
+
+- Add transaction fields to the `getblocktemplate` RPC ([#5496](https://github.com/ZcashFoundation/zebra/pull/5496) and [#5508](https://github.com/ZcashFoundation/zebra/pull/5508))
+- Implement coinbase conversion to RPC `TransactionTemplate` type ([#5554](https://github.com/ZcashFoundation/zebra/pull/5554))
+
+### Fixed
+
+- Return an error instead of panicking in the batch verifier on shutdown ([#5530](https://github.com/ZcashFoundation/zebra/pull/5530))
```

Comment:
```suggestion
- Check network and P2SH addresses for mining config and funding streams([#5620](https://github.com/ZcashFoundation/zebra/pull/5620))
- Return an error instead of panicking in the batch verifier on shutdown ([#5530](https://github.com/ZcashFoundation/zebra/pull/5530))
```